lib/process-services/src/lib/form/widgets/dropdown/dropdown.widget.ts

```diff
@@ -119,7 +124,15 @@ export class DropdownWidgetComponent extends WidgetComponent implements OnInit {
         return this.field.type === 'readonly';
     }
 
+    private isNoneValueSelected(value: string): boolean {
+        return value === undefined;
```

Comment:
are you sure this will work to show the required error when the none option is selected? In my fix for APA I did that because we changed the way the dropdown widgets work to match the Angular Material approach https://material.angular.io/components/select/overview#select-hint-error . 

With that approach when the none option is selected the value equals undefined. As far as I can understand from seeing this code I don't see a similar implementation for that, so your none value still contains the `id: 'empty'`. Considering that I don't think this method is what you need.

Have you tested it and it works?